src/TLuaInterpreter.cpp

```diff
@@ -2971,6 +2971,35 @@ int TLuaInterpreter::setModulePriority(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::getModuleSync(lua_State* L)
+{
+  QString moduleName;
+  if (!lua_isstring(L, 1)) {
+    lua_pushstring(L, "getModuleSync: Module must be a string");
+    return lua_error(L);;
+  } else {
+    moduleName = lua_tostring(L, 1);
```

Comment:
Remember the else-after-return anti-pattern, and that the moduleName is not confined to ASCII characters, and the standard format of our error messages about argument types (and that `;;` does not do anything more than `;`) - I'd recommend changing:
```cpp
    QString moduleName;
    if (!lua_isstring(L, 1)) {
        lua_pushstring(L, "getModuleSync: Module must be a string");
        return lua_error(L);;
    } else {
        moduleName = lua_tostring(L, 1);
    }
```
to:
```cpp
    if (!lua_isstring(L, 1)) {
        lua_pushfstring(L, "getModuleSync: bad argument #1 type (module name as string expected, got %s!)", luaL_typename(L, 1));
        return lua_error(L);
    }
    QString moduleName = QString::fromUtf8(lua_tostring(L, 1));
```

Reply:
I honestly do not know how two semicolons ended up there. I'm glad your eyes are sharper than mine!

Reply:
Do you write a lot of `bash` *shell-scripts* perhaps? :laughing: